openhtf/io/user_input.py

```diff
@@ -63,7 +61,8 @@ def __init__(self):
     self._response = None
     self._cond = threading.Condition()
 
-  def DisplayPrompt(self, message, text_input=False):
+  def DisplayPrompt(self, message, text_input=False,
+                    timeout=int(FLAGS.prompt_timeout_s)):
```

Comment:
This might be my fault from before, but the int() cast probably isn't needed here since this flag is declared as an integer. I wonder why I put it in.